src/store/modules/sitestatus/getters/wema_settings_getters.js

```diff
@@ -29,9 +29,19 @@ const owmActive = (state, getters) => {
 }
 
 const observingMode = (state, getters) => {
+  let observingModeDisplay = state.wema_settings.observing_mode ?? 'n/a'
```

Comment:
He also talked to me about adding 'Engineering' as the display and sending the string 'engineering' - though his code cannot yet recognize it.

Reply:
This part is actually just for handling the status display, specifically translating between 'active'<-->'Online' and 'inactive'<-->'Offline'. 

The 'engineering' status doesn't need this and so it's not included here, but I added it as an option for the user to select in Enclosure.vue